cmd/lncli/commands.go

```diff
@@ -418,6 +431,15 @@ func sendCoins(ctx *cli.Context) error {
 		displayAmt = balanceResponse.GetConfirmedBalance()
 	}
 
+	if ctx.IsSet("utxo") {
```

Comment:
I think we need to prevent invalid combinations here and only call the rpc if it makes sense. I think rn it is possible to pass in `sweepall` and `utxo` as arguments at the same time which doesn't make sense.